L1Trigger/L1TCommon/python/customsPostLS1.py

```diff
@@ -78,9 +78,19 @@ def customiseSimL1EmulatorForStage1(process):
 
 from L1Trigger.Configuration.customise_overwriteL1Menu import *
 
+def customiseSimL1EmulatorForPostLS1_lowPU(process):
+    # move to the lowPU v2 L1 menu once the HLT has been updated accordingly
+    process = L1Menu_Collisions2015_lowPU_v2(process)
+    return process
+
 def customiseSimL1EmulatorForPostLS1_50ns(process):
-    # move to the 50ns v0 L1 menu once the HLT has been updated accordingly
-    process = L1Menu_Collisions2015_50ns_v1(process)
+    # move to the 50ns v2 L1 menu once the HLT has been updated accordingly
+    process = L1Menu_Collisions2015_50ns_v2(process)
```

Comment:
In principle it does change the MC.
In practice, it only adds new L1 seeds, so it should not be a problem:
```diff
--- L1Menu_Collisions2015_50nsGct_v1    2015-05-28 09:08:48.193133942 +0200
+++ L1Menu_Collisions2015_50nsGct_v2    2015-05-28 09:08:51.385133895 +0200
@@ -2,6 +2,10 @@
    1                                                       L1_AlwaysTrue                                                       L1_AlwaysTrue
    2                                                        L1_SingleEG2                                                        L1_SingleEG2
    3                                                      L1_SingleJet16                                                      L1_SingleJet16
+   4                                            L1_MinimumBiasHF1_AND_v1                                               L1_MinimumBiasHF1_AND
+   5                                                L1_MinimumBiasHF1_OR                                                L1_MinimumBiasHF1_OR
+   6                                               L1_MinimumBiasHF2_AND                                               L1_MinimumBiasHF2_AND
+   7                                                L1_MinimumBiasHF2_OR                                                L1_MinimumBiasHF2_OR
    8                                    L1_Mu16er_TauJet40erORCenJet72er                                    L1_Mu16er_TauJet40erORCenJet72er
    9                                                        L1_Mu12_EG10                                                        L1_Mu12_EG10
   10                                                         L1_Mu5_EG15                                                         L1_Mu5_EG15
@@ -22,6 +26,7 @@
   25                                                 L1_DoubleTauJet44er                                                 L1_DoubleTauJet44er
   26                                                        L1_DoubleMu0                                                        L1_DoubleMu0
   27                                                     L1_SingleMu30er                                                     L1_SingleMu30er
+  28                                                            L1_ETT15                                                            L1_ETT15
   29                                             L1_Mu3_JetC16_WdEtaPhi2                                             L1_Mu3_JetC16_WdEtaPhi2
   30                                             L1_Mu3_JetC52_WdEtaPhi2                                             L1_Mu3_JetC52_WdEtaPhi2
   31                                                    L1_EG25er_HTT125                                                    L1_EG25er_HTT125
@@ -52,6 +57,7 @@
   56                                                       L1_SingleMu16                                                       L1_SingleMu16
   57                                                       L1_SingleEG15                                                       L1_SingleEG15
   58                                                        L1_SingleMu5                                                        L1_SingleMu5
+  59                                                            L1_ETT40                                                            L1_ETT40
   60                                                     L1_SingleMu20er                                                     L1_SingleMu20er
   61                                                       L1_SingleMu12                                                       L1_SingleMu12
   62                                                       L1_SingleMu20                                                       L1_SingleMu20
@@ -86,8 +92,12 @@
   91                                                    L1_Mu5_DoubleEG5                                                    L1_Mu5_DoubleEG5
   92                                                 L1_DoubleEG6_HTT150                                                 L1_DoubleEG6_HTT150
   93                                              L1_QuadJetC36_TauJet52                                              L1_QuadJetC36_TauJet52
+  94                                                            L1_ETT60                                                            L1_ETT60
+  95                                                            L1_ETT90                                                            L1_ETT90
+  96                                                     L1_DoubleMuOpen                                                     L1_DoubleMuOpen
   97                                                        L1_TripleMu0                                                        L1_TripleMu0
   98                                                   L1_TripleMu_5_5_3                                                   L1_TripleMu_5_5_3
+  99                                                      L1_SingleJet12                                                      L1_SingleJet12
  100                                                 L1_TripleEG_14_10_8                                                 L1_TripleEG_14_10_8
  101                                                   L1_DoubleEG_15_10                                                   L1_DoubleEG_15_10
  102                                                   L1_DoubleEG_22_10                                                   L1_DoubleEG_22_10
@@ -105,6 +115,8 @@
  114                                         L1_DoubleMu0_Eta1p6_WdEta18                                         L1_DoubleMu0_Eta1p6_WdEta18
  115                                             L1_ETM60_NotJet52WdPhi2                                             L1_ETM60_NotJet52WdPhi2
  116                                             L1_ETM70_NotJet52WdPhi2                                             L1_ETM70_NotJet52WdPhi2
+ 117                                                           L1_ETT130                                                           L1_ETT130
+ 118                                                     L1_RomanPotsAND                                                     L1_RomanPotsAND
  119                                               L1_TripleJet_92_76_64                                               L1_TripleJet_92_76_64
  121                                                          L1_QuadMu0                                                          L1_QuadMu0
  122                                                     L1_SingleMu18er                                                     L1_SingleMu18er
```